### PR TITLE
adds create action

### PR DESCRIPTION
**Pull request by dougperez**
this add create action, also adds some shared examples.

### CODE REVIEW

File: lib/conduit/cogotech/actions/base.rb [hunk 1]
```diff
@@ -14,7 +14,21 @@ def http_method(http_method = nil)
     end
 
     def remote_url
-      "http://#{production_url}"
+      "http://#{production_url}" + action_url
```
Comment (pedromartinez) [inline lib/conduit/cogotech/actions/base.rb]: Interpolation here?

File: lib/conduit/cogotech/actions/create.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'conduit/cogotech/actions/base'
+
+module Conduit::Driver::Cogotech
+  class Create < Conduit::Driver::Cogotech::Base
```
Comment (pedromartinez) [inline lib/conduit/cogotech/actions/create.rb]: CreateDevice?
AddDevice?

Comment (dougperez) [inline lib/conduit/cogotech/actions/create.rb]: I kept going back and forth from create vs create device. Conduit-sprint has activate/suspend/deactivate, is the reason I just went with create. Do you have a preference?

File: lib/conduit/cogotech/actions/base.rb (hunk 1 above)
Comment (dougperez) [inline lib/conduit/cogotech/actions/base.rb]: Yup.

File: lib/conduit/cogotech/actions/create.rb (hunk 2 above)
Comment (pedromartinez) [inline lib/conduit/cogotech/actions/create.rb]: How about calling it activate?

Comment (dougperez) [inline lib/conduit/cogotech/actions/create.rb]: Cogotech already has a activate endpoint.

Comment (pedromartinez) [inline lib/conduit/cogotech/actions/create.rb]: Yeah but I think that endpoint acts more like a restore.

Comment (dougperez) [inline lib/conduit/cogotech/actions/create.rb]: Ah, ok yeah I think activate is fine.

Comment (dougperez) [inline lib/conduit/cogotech/actions/create.rb]: Renamed to activate.

**Comment by pedromartinez** [comment]
👍

**Pull request merged**